app/src/pages/inside/logsPage/logToolbar/logToolbar.jsx

```diff
@@ -149,6 +161,17 @@ export class LogToolbar extends Component {
           allEventClick={LOG_PAGE_EVENTS.ALL_LABEL_BREADCRUMB}
           onRestorePath={restorePath}
         />
+        <InputCheckbox
+          onChange={() => {
+            !this.state.isChecked
```

Comment:
Rename `isChecked` according to its meaning.

Reply:
Changed in EPMRPP-59880